02week/pigLatin.js

```diff
@@ -3,49 +3,136 @@
 const assert = require('assert');
 const readline = require('readline');
 const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
 
 
 function pigLatin(word) {
 
-  // Your code here
+    // Your code here
+    const arrayVowels = ['a', 'e', 'i', 'o', 'u'];
+    const arrayConsonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'x', 'y', 'z'];
```

Comment:
This is unnecessary, you can simplify this a lot. How about checking if every word is a letter then just checking if it's a vowel.